src/components/UserEmail/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function UserEmail() {
```

Comment:
UserEmail what? UserPassword what? i'm not sure why both of those require their own component actually

Reply:
I have just finished them. These component are huge, so I put them in a separate component.

UserDetail screenshot:
![image](https://user-images.githubusercontent.com/30551053/99296233-77140a80-2857-11eb-9779-02ff41109e54.png)

UserEmail screenshot:
![image](https://user-images.githubusercontent.com/30551053/99296278-8b580780-2857-11eb-8b3c-777d6888d81e.png)
The last one also divided into two sub-components